docs/source/topics/proc_installing-codeready-containers.adoc

```diff
@@ -1,11 +1,57 @@
 [id="installing-codeready-containers_{context}"]
 = Installing {prod}
 
+{prod} is available as a portable executable for {rhel} and {msw}.
+On {mac}, {prod} is available using a guided installer.
+
 .Prerequisites
 
 * Your host machine must meet the minimum system requirements.
 For more information, see link:{crc-gsg-url}#minimum-system-requirements_gsg[Minimum system requirements].
 
 .Procedure
 
-. Download the link:{crc-download-url}[latest release of {prod}] for your platform and extract the contents of the archive to a location in your `_PATH_`.
+. Download the link:{crc-download-url}[latest release of {prod}] for your platform.
```

Comment:
The linked page takes me to a 404 error page. See the screenshot: 

<img width="1233" alt="Screen Shot 2021-06-09 at 10 43 10 AM" src="https://user-images.githubusercontent.com/70717303/121376401-7f28ba00-c90f-11eb-8571-6df4d110ec17.png">

Reply:
The URL is an attribute, so GitHub may not know how to render it correctly. The link should take you to this page when properly rendered: https://cloud.redhat.com/openshift/create/local

For the record, you can also generate the documentation locally by running `make build_docs` in the repository. The generated docs are in `docs/build/index.html`. :)